### PR TITLE
Add write_to_firestore example

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

File: docs/write_to_firestore.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Example 6: Send model data to Google Firestore
+Starting version 0.1.30, fal lets you easily send data to [Google Firestore](https://cloud.google.com/firestore/). This example assumes that Firestore is [already enabled](https://cloud.google.com/firestore/docs/quickstart-servers#create_a_in_native_mode_database) in your Google Cloud project.
+
+## Credentials
+By default fal will attempt to use the credentials provided in profiles.yml. If that's unsuccessful, credentials environment variable is checked. See [here](https://cloud.google.com/docs/authentication/getting-started) for more information on how to set such environment variable.
```
Comment (chamini2) [inline docs/write_to_firestore.md:5]: great

File: docs/write_to_firestore.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  fal:
+    scripts:
+      - path_to_fal_script.py
+```
+
+## `write_to_firestore` magic function
+This function is available in fal runtime. There is no need to import it. Here's an example invocation:
+
+```python
+write_to_firestore(df=dataframe, collection=collection_name, key_column=key_column)
```
Comment (chamini2) [inline docs/write_to_firestore.md:20]: You know we can make naming the parameters mandatory by changing `write_to_firestore` function definition. Do we want that?

**Review by chamini2 — APPROVED** [review]
👍

File: docs/write_to_firestore.md (hunk 2 above)
Comment (mkam039) [reply docs/write_to_firestore.md:20]: It's ok to give users flexibility here.

**Pull request merged**